src/main/resources/application.yml

```
@@ -27,4 +27,8 @@ server:
     encoding:
       charset: UTF-8   # 문자 인코딩 UTF-8로 설정 (한글 깨짐 방지)
       enabled: true    # 인코딩 설정 켜기
-      force: true      # 모든 요청/응답에 UTF-8 강제로 적용
\ No newline at end of file
+      force: true      # 모든 요청/응답에 UTF-8 강제로 적용
+
+openweather:
+  api:
+    key: 2b6b34b581235fe0d0e43913fd2b4dfb
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)